pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -97,9 +97,20 @@ func (r *Reconciler) gatherSelectedResource(placement string, selectors []fleetv
 	}
 	// sort the resources in strict order so that we will get the stable list of manifest so that
 	// the generated work object doesn't change between reconcile loops
+	sortResources(resources)
+
+	return resources, nil
+}
+func sortResources(resources []runtime.Object) {
 	sort.Slice(resources, func(i, j int) bool {
 		obj1 := resources[i].DeepCopyObject().(*unstructured.Unstructured)
 		obj2 := resources[j].DeepCopyObject().(*unstructured.Unstructured)
+		if obj1.GetObjectKind().GroupVersionKind().String() == utils.NamespaceGVK.String() || obj2.GetObjectKind().GroupVersionKind().String() == utils.NamespaceGVK.String() {
```

Comment:
Isn't this renamed to utils.NamespaceMetaGVK? Where is NamespaceGVK defined?

Reply:
NamespaceGVK is defined here https://github.com/Azure/fleet/blob/127e5d65b37d905113d3438babdf1c3952dfb84b/pkg/utils/common.go#L125. All the schema.GroupVersioKind are grouped together while metav1.GroupVersionKind here https://github.com/Azure/fleet/blob/127e5d65b37d905113d3438babdf1c3952dfb84b/pkg/utils/common.go#L174. Would you rather have them combined?